### ISSUE TITLE
[Bug] Ignored videos being deleted

### ISSUE THREAD

**Issue by rssnyder** [jlesage/docker-handbrake#274]
### Current Behavior

When a video file that is not in the automated transcode extension list is scanned, it is deleted.

### Expected Behavior

Videos not of the target type are left alone.

### Steps To Reproduce

run handbrake docker with non ts files in the folder and when scanned and ignore they are deleted

### Environment

- OS: ubuntu server
- OS version: 22.04
- CPU: Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
- Docker version: Docker version 20.10.22, build 3a2c30b
- Device model: whitebox
- Browser/OS: not used

### Container creation

```yaml
---
version: "3"
services:
  handbreak:
    image: jlesage/handbrake
    container_name: handbrake
    ports:
      - 5800:5800
    volumes:
      - $APPDATA/handbreak:/config
      - /bucket/media/tv:/watch
      - /bucket/media/tv:/output
    environment:
      USER_ID: 65534 
      GROUP_ID: 1001
      AUTOMATED_CONVERSION: 1
      AUTOMATED_CONVERSION_PRESET: "GENERAL/Fast 1080p30"
      AUTOMATED_CONVERSION_KEEP_SOURCE: 0
      AUTOMATED_CONVERSION_OUTPUT_SUBDIR: SAME_AS_SRC
      AUTOMATED_CONVERSION_VIDEO_FILE_EXTENSIONS: "ts"
```

### Container log

```text
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S26/S26E07.mp4' (abec75737bd9a9fedd280f86dd0d3237) has an extension not part of the inclusion list.
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S26/S26E07.mp4' (abec75737bd9a9fedd280f86dd0d3237) is not a video, ignoring...
[autovideoconverter] Removed /watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S26/S26E07.mp4'.
[autovideoconverter] Removed directory '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S26'.
[autovideoconverter] Waiting 5 seconds before processing '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S30/S30E11.mp4'...
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S30/S30E11.mp4' (2fcf658fbcefd0b6090283bac4650e2f) has an extension not part of the inclusion list.
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S30/S30E11.mp4' (2fcf658fbcefd0b6090283bac4650e2f) is not a video, ignoring...
[autovideoconverter] Removed /watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S30/S30E11.mp4'.
[autovideoconverter] Removed directory '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S30'.
[autovideoconverter] Waiting 5 seconds before processing '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S20/S20E09.mp4'...
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S20/S20E09.mp4' (1c967ee432642bf9e0ca743b1ec33410) has an extension not part of the inclusion list.
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S20/S20E09.mp4' (1c967ee432642bf9e0ca743b1ec33410) is not a video, ignoring...
[autovideoconverter] Removed /watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S20/S20E09.mp4'.
[autovideoconverter] Removed directory '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S20'.
[autovideoconverter] Waiting 5 seconds before processing '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S29/S29E02.mp4'...
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S29/S29E02.mp4' (fd877e01786e45437d103fd45c2ba275) has an extension not part of the inclusion list.
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S29/S29E02.mp4' (fd877e01786e45437d103fd45c2ba275) is not a video, ignoring...
[autovideoconverter] Removed /watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S29/S29E02.mp4'.
[autovideoconverter] Removed directory '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S29'.
[autovideoconverter] Waiting 5 seconds before processing '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S24/S24E14.mp4'...
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S24/S24E14.mp4' (b1b089b156f70b49dd16eee17b0a271d) has an extension not part of the inclusion list.
[autovideoconverter] File '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S24/S24E14.mp4' (b1b089b156f70b49dd16eee17b0a271d) is not a video, ignoring...
[autovideoconverter] Removed /watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S24/S24E14.mp4'.
[autovideoconverter] Removed directory '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD/S24'.
[autovideoconverter] Removed directory '/watch/Survivor/Survivor.S16-S30.1080p.WEBRip.DDP5.1.x264-MiXeD'.
```

**Comment by jlesage** [comment]
The file is removed because `AUTOMATED_CONVERSION_KEEP_SOURCE` is set to `0`.

Also, I see that both `/output` and `/watch` are mapped to the same folder on the host.  Note that the automatic video converter has not been designed for this scenario.  It's probably better to have a separate watch folder and let the automatic video converter move the file to its final location once the conversion is done.

**Comment by jlesage** [comment]
@rssnyder, do you have any update on this ?
